modules/stanford_events_importer/stanford_events_importer.module

```diff
@@ -0,0 +1,133 @@
+<?php
+
+/**
+ * @file
+ * File description.
+ *
+ * Long description.
+ */
+
+use Drupal\migrate_tools\MigrateBatchExecutable;
+use Drupal\migrate\MigrateMessage;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\user\Entity\User;
+use Drupal\stanford_events_importer\StanfordEventsImporter;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Alter the config form to add the migrate_tools UI.
+ */
+function stanford_events_importer_form_config_pages_stanford_events_importer_form_alter(array &$form, FormStateInterface $form_state) {
+  $user = User::load(\Drupal::currentUser()->id());
+  $form['actions']['#type'] = 'fieldset';
+
+  $form['actions']['import'] = [
+    '#type' => 'submit',
+    '#value' => t('Save & Import'),
+    '#name' => 'op',
+    '#button_type' => "primary",
+    '#submit' => [
+      "::submitForm",
+      "::save",
+      "stanford_events_importer_form_config_pages_stanford_events_importer_form_alter_submit",
```

Comment:
holy name. what might be easier is to have the submit handler in the service. :shrug:

Reply:
I suppose I could choose any name here but [module_name][form_ID_form_alter][suffix] is clear to what this does even if it is terribly long.